experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -174,65 +132,67 @@ export interface MoveOut<TNodeChange = NodeChangeType>
 	extends HasRevisionTag,
 		HasMoveId,
 		HasChanges<TNodeChange>,
-		CanConflict {
+		CellTargetingMark {
 	type: "MoveOut";
 	count: NodeCount;
-	/**
-	 * When true, the corresponding MoveIn has a conflict.
-	 * This is independent of whether this mark has a conflict.
-	 */
-	isDstConflicted?: true;
 }
 
 export interface HasReattachFields extends HasPlaceFields {
-	/**
-	 * The tag of the change that detached the data being reattached.
-	 *
-	 * Undefined when the reattach is the product of a tag-less change being inverted.
-	 * It is invalid to try convert such a reattach mark to a delta.
-	 */
-	detachedBy: RevisionTag | undefined;
+	inverseOf?: RevisionTag;
```

Comment:
Add comment explaining that when this is populated the change is a revert-only inverse.